### ISSUE TITLE
Licensing for commercial use - how to contact you?

### ISSUE THREAD

**Issue by petrsmid** [Deci-AI/super-gradients#2067]
### 💡 Your Question

Hello,
There are multiple LICENSE files in your repository and it is unclear to me which license applies. Two files state that if we want to use your code commercially we should contact you. However, your web page does not exist anymore and we cannot find any way how to contact you. Could you please advise us?

Thank you,
Petr

**Comment by nkinnaird** [comment]
I would also like to know the answer to this question.

**Comment by CrasCris** [comment]
it may be in stand by cuz NVIDIA bought Deci-AI

**Comment by nkinnaird** [comment]
That's definitely the reason. I hope that Nvidia fully open sources this project, but not sure, so good to leave this question up in case anyone ever comes back to answer it.

**Comment by Gabriellgpc** [comment]
I also I'd like to know about that 👀

**Comment by fperezgamonal** [comment]
I'm also interested in clarification regarding the license. Hopefully NVIDIA open sources the entire project and we don't end up like we did with Ultralytics, close sourcing everything, including the pre-trained weights (which most people will severely alter through fine-tuning any way...).

**Comment by senovr** [comment]
I think it is better to reach founders of Deci via other channels- such as linkedin.
Here is startup info:
https://finder.startupnationcentral.org/company_page/deci-ai?section=team
